doc/reference/devices_gpu.md

```diff
@@ -51,8 +51,105 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
     lxc config device add <instance_name> <device_name> gpu gputype=physical pci=<pci_address>
 
+Add a specific GPU from the host system as a `physical` GPU device to an instance using the [Container Device Interface](https://github.com/cncf-tags/container-device-interface) (CDI) notation.
+
+    lxc config device add <instance_name> <device_name> gpu gputype=physical id=<fq_CDI_name>
+
 See {ref}`instances-configure-devices` for more information.
 
+#### Passing an NVIDIA iGPU to a container
+
+If you have NVIDIA runtime libraries and configuration installed on your host and you want to pass these files to your container, you should add a device with the CDI notation.
+Consider the following scenario:
+
+Your host is an NVIDIA single board computer that has a Tegra SoC with an iGPU, and you have an SDK for handling AI workloads installed on the host. You want to create a LXD container and run an inference job inside the container using the iGPU as a backend. This job should run inside a Docker container (or another OCI-compliant runtime).
+To achieve this, complete the following steps:
+
+1. Initialize a LXD container:
+
+       lxc init ubuntu:24.04 t1 --config security.nesting=true --config security.privileged=true
```

Comment:
I don't think we should be encouraging the use of security.privileged in this case.

What was the requirement for needing privileged?

Reply:
> Docker requires a privileged LXD container in order to run properly.

This isn't entirely true. Do you know why it needs privileged?

It is possible to run some docker containers without privileged mode, so we should seek to understand what in this specific example requires it.

I think we should use the simple example you mention here, and then move the more detail specific example into a howto/tutorial page.

Reply:
fixed